### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by BWANG2016** [martinpacesa/BindCraft#260]
Hello,

I was trying to run the PDL1 example, but the execution was terminated due to a segmentation fault after stage1: test logits. 

python -u ./bindcraft.py --settings './settings_target/PDL1.json' --filters './settings_filters/default_filters.json' --advanced './settings_advanced/default_4stage_multimer.json'

Available GPUs:
NVIDIA RTX A60001: gpu
NVIDIA RTX A60002: gpu
┌──────────────────────────────────────────────────────────────────────────────┐
│                                 PyRosetta-4                                  │
│              Created in JHU by Sergey Lyskov and PyRosetta Team              │
│              (C) Copyright Rosetta Commons Member Institutions               │
│                                                                              │
│ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE │
│         See LICENSE.PyRosetta.md or email license@uw.edu for details         │
└──────────────────────────────────────────────────────────────────────────────┘
PyRosetta-4 2024 [Rosetta PyRosetta4.Release.python310.linux 2024.38+release.200d5f9a7d8cdd7afdd078f156da6b5a7d97543f 2024-09-11T17:31:36] retrieved from: http://www.pyrosetta.org
Running binder design for target PDL1
Design settings used: default_4stage_multimer
Filtering designs based on default_filters
Starting trajectory: PDL1_l122_s800100
Stage 1: Test Logits
Segmentation fault

Thanks
